test/highlight/elixir/regular.exs

```diff
@@ -22,6 +22,10 @@ defmodule Regular do
         IO.puts "The first value is #{head}"
         head + (((1 + (2 + 3))))
     end
+
+    defp accessLookup(map) do
```

Comment:
Don't you need `x` as a function parameter?

```suggestion
    defp accessLookup(map, x) do
```

My Elixir is a bit rusty, so please double-check. The code does not have to compile, but it should if possible.